src/applications/simple-forms/21-4142/pages/authorization.js

```diff
@@ -0,0 +1,117 @@
+          </p>
+          <p>
+            3. Information created within 12 months after the date this
+            authorization is signed in Item 11, as well as past information.
+          </p>
+          <p>
+            You should not complete this form unless you want the VA to obtain
+            private treatment records on your behalf. If you have already
+            provided these records or intend to obtain them yourself, there is
+            no need to fill out this form. Doing so will lengthen your claim
+            processing time.
+          </p>
+          <h4 className="vads-u-color--gray-dark">Important</h4>
+          <p>
+            In accordance with 38 C.F.R. §3.159(c), "VA will not pay any fees
+            charged by a custodian to provide records requested."
+          </p>
+          <h4 className="vads-u-color--gray-dark">Acknowledgment</h4>
+          <p>
+            I hereby authorize the sources listed in Section IV, to release any
+            information that may have been obtained in connection with a
+            physical, psychological or psychiatric examination or treatment,
+            with the understanding that VA will use this information in
+            determining my eligibility to veterans benefits I have claimed.
+          </p>
+          <p>
+            I understand that the source being asked to provide the Veterans
+            Benefits Administration with records under this authorization may
+            not require me to execute this authorization before it provides me
+            with treatment, payment for health care, enrollment in a health
+            plan, or eligibility for benefits provided by it.
+          </p>
+          <p>
+            I understand that once my source sends this information to VA under
+            this authorization, the information will no longer be protected by
+            the HIPAA Privacy Rule, but will be protected by the Federal Privacy
+            Act, 5 USC 552a, and VA may disclose this information as authorized
+            by law.
+          </p>
+          <p>
+            I also understand that I may revoke this authorization in writing,
+            at any time except to the extent a source of information has already
+            relied on it to take an action. To revoke, I must send a written
+            statement to the VA Regional Office handling my claim or the Board
+            of Veterans' Appeals (if my claim is related to an appeal) and also
+            send a copy directly to any of my sources that I no longer wish to
+            disclose information about me.
+          </p>
+          <p>
+            I understand that VA may use information disclosed prior to
+            revocation to decide my claim.
+          </p>
+        </>
+      ),
+    },
+    acknowledgeToReleaseInformation: {
+      'ui:title': 'I acknowledge and authorize this release of information',
+      'ui:required': () => true,
```

Comment:
Fairly minor, but I don't think you need this line because you also put `acknowledgeToReleaseInformation` in the `required` array in the schema below.